WooCommerce/UITestsFoundation/XCTest+Extensions.swift

```diff
@@ -114,8 +114,44 @@ extension XCUIElement {
         startCoordinate.press(forDuration: 0.01, thenDragTo: destination)
     }
 
-    public func getTextVisibilityCount(text: String) throws -> Int {
-        let predicate = NSPredicate(format: "label CONTAINS[c] %@", text)
+    func getStaticTextVisibilityCount(textToFind: String) throws -> Int {
+        let predicate = NSPredicate(format: "label CONTAINS[c] %@", textToFind)
         return XCUIApplication().staticTexts.containing(predicate).count
     }
+
+    public func assertTextVisibilityCount(textToFind: String) {
+        let count = try! getStaticTextVisibilityCount(textToFind: textToFind)
+        XCTAssertTrue(count == 1, "Expected '\(textToFind)' to appear once, but it appeared '\(count)' times!")
+    }
```

Comment:
The function name and the argument it receives does not indicate how many times the text is expected to appear.
What do you think about either renaming the function to something like `assertTextVisibleOnce(textToFind)` or adding another argument like `assertTextVisibilityCount(textToFind: String, expectedCount: Int)`?

Reply:
Makes sense, I'll update this in the next PR (along with other changes).